infra/README.md

```diff
@@ -72,6 +72,7 @@ There are some parts of this project's infrastructure that are **not** managed/c
 - Certificates in Certificate Manager
 - AWS Amplify config
 - Cloudfront distributions that sit between API domains (e.g. int.api.octopus.ac) and the corresponding API gateway
+- An IAM identity provider, with an associated IAM role, to allow github actions to act on AWS
```

Comment:
I didn't do any of this in terraform because the IAM provider needed admin rights and I didn't have time to mess with terraform.

Is the role something you could potentially add to terraform?

Reply:
Seems to work terraforming the role!